src/scribe_data/cli/list.py

```diff
@@ -0,0 +1,153 @@
+import json
+from pathlib import Path
+from typing import Dict, List, Union
+
+# Load language metadata from JSON file
+METADATA_FILE = Path(__file__).parent.parent / 'resources' / 'language_meta_data.json'
+
+def load_language_metadata() -> Dict:
+    with METADATA_FILE.open('r', encoding='utf-8') as file:
+        return json.load(file)
+
+LANGUAGE_METADATA = load_language_metadata()
```

Comment:
I'm going to switch these to just normal snake case as to me "screaming snake case" is for constants that are defined directly, not function results. `METADATA_FILE` above is generally ok, but generally I'd consider:

- `A_STRING = "string"`
- `A_NEW_STRING = f"new {A_STRING}"`
- `AN_INT = 5`
- etc

The `METADATA_FILE` path should not change, but then we're going to be expanding `language_metadata` from time to time.

Let  me know if this makes sense, @mhmohona and @wkyoshida!